api/doc.go

```diff
@@ -0,0 +1,51 @@
+// Package api is the topmost API interface package for users using sentinel-golang(aka sentinel)
+//
+// Users must initialize sentinel before loading sentinel rules. Sentinel support three ways to perform initialization:
+//
+//  1. api.InitDefault(), using default config to initialize;
+//  2. api.InitWithConfig(confEntity *config.Entity), using customized config Entity to initialize
+//  3. api.InitWithConfigFile(configPath string), using yaml file to initialize.
+//
+// Here is the example code to use sentinel:
+//
+//  import sentinel "github.com/alibaba/sentinel-golang/api"
+//
+//  err := sentinel.InitDefault()
+//  if err != nil {
+//      log.Fatal(err)
+//  }
+//
+//  //Load sentinel rules
```

Comment:
The example might be too complicated? Actually we may introduce APIs in this package.